src/chainparams.cpp

```diff
@@ -599,6 +599,50 @@ class CTestNetParams : public CChainParams {
                 Consensus::FS_DEFERRED,
                 nu6Activation,
                 3396000);
+
+            // ZIP 214 Revision 2
+            // FPF uses a single address repeated 27 times, once for each funding period.
+            // There are 27 periods because the start height is after the second halving
+            // on testnet and does not align with a period boundary.
+            std::vector<std::string> fpf_addresses_h3(27, "t2HifwjUj9uyxr9bknR8LFuQbc98c3vkXtu");
```

Comment:
I have verified that 27 intervals will cover the 939500 block period between NU6.1 activation and the given end height, given the funding period of 35,000 blocks (840000 * 2 / 48).